skimage/registration/_phase_cross_correlation.py

```diff
@@ -136,13 +136,13 @@ def _disambiguate_shift(reference_image, moving_image, shift):
     moving_image : numpy array
         The moving image: applying the shift to this image overlays it on the
         reference image. Must be the same shape as the reference image.
-    shift : tuple of float
+    shift : ndarray
         The shift to apply to each axis of the moving image, *modulo* image
         size. The length of ``shift`` must be equal to ``moving_image.ndim``.
 
     Returns
     -------
-    real_shift : tuple of float
+    real_shift : sequence of floats
```

Comment:
@mkcor, agreed. This line should be

```suggestion
    real_shift : ndarray
```